### PR TITLE
Minor bugfixes

### PR DESCRIPTION
**Pull request by nobodywasishere**
- Only support `file` and `untitled` schemas
- Need to pass filename via `--stdin-filename` to support ameba config excludes 
- Minor logging fixes around untitled documents

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -157,9 +176,9 @@ export function noWorkspaceFolder(uri: Uri): WorkspaceFolder {
 }
 
 export function isCrystalDocument(doc: TextDocument): boolean {
-    return doc.languageId === 'crystal'
+    return doc.languageId === 'crystal' && (doc.uri.scheme === 'file' || doc.uri.scheme === 'untitled')
```
Comment (Sija) [inline src/extension.ts:179]: What's the reason of this change? Also, that's not what the name of the function suggest.

Comment (nobodywasishere) [reply src/extension.ts:179]: We only want to support these two schemas, as otherwise it creates a mess locally (looking at git changes, will duplicate all the lints, which is annoying).

Comment (Sija) [reply src/extension.ts:179]: So what's the reason for the _mess_?

Comment (nobodywasishere) [reply src/extension.ts:179]: Feel free to make any commits to this PR to get it exactly how you want it.

Comment (Sija) [reply src/extension.ts:179]: Thanks, I'm trying to understand the reasoning behind these changes.

Comment (nobodywasishere) [reply src/extension.ts:179]: Like I said, it's to only support the file and untitled schemas. I don't see a problem with limiting the schema checks within this method, as otherwise would require adding an additional method for checking the schema to everywhere this method is already called. This method was also originally called something to the effect of "isValidCrystalDocument" until y'all had me change it.

**Review by veelenga — APPROVED** [review]
lgtm

**Review by Sija — APPROVED** [review]
